oak-blob-cloud-azure/pom.xml

```diff
@@ -191,10 +191,10 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
-
         <dependency>
-            <groupId>com.arakelian</groupId>
-            <artifactId>docker-junit-rule</artifactId>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
+            <version>${testcontainers.version}</version>
```

Comment:
Why do we need the bersion property?

Reply:
defining a version is always beneficial, test containers is widely used and it might pick up some different version from some other modules which might lead to version mismatch. In this case it will always refer oak-parent for test container version

Reply:
I don't get that. We define the dependency with version in the parent POM, and then use it version-less in all subprojects. That's IMHO the right approach, and no variable is needed at all.

Reply:
Yes we do that, but to be sure that other modules inherit the same version of dependency that is declared in parent module and does not transitively use some other version of same dependency, it will always beneficial to define the version in child modules to avoid dependency conflicts.

https://stackoverflow.com/questions/13074231/override-library-version-defined-in-parent-pom

Reply:
I still don't get it.

If you believe that we have a general problem with how we maintain version numbers of dependencies in oak-parent, please open a separate ticket and explain the issue - for visibility and so that we can fix the problem systematically.

Otherwise, please just stick with the project conventions.

Reply:
following the project conventions, removed test container dependency from child modules.